doc/rados/configuration/mon-config-ref.rst

```diff
@@ -591,7 +591,7 @@ Trimming requires that the placement groups are ``active + clean``.
 
 ``paxos trim disabled max versions``
 
-:Description: The maximimum number of version allowed to pass without trimming.
+:Description: The maximum number of version allowed to pass without trimming.
```

Comment:
Hi, @tchaikov 

Thanks for your suggestion,
I will close this PR and create new one to remove those options.